files/__addonLocation__/babel.config.json

```diff
@@ -1,9 +1,9 @@
 {
 <% if (typescript) { %>  "presets": [["@babel/preset-typescript"]],
-<% } %>  "plugins": [<% if (typescript) { %>
+<% } %>  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",<% if (typescript) { %>
     ["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],<% } %>
-    "@embroider/addon-dev/template-colocation-plugin",
-    ["@babel/plugin-proposal-decorators", { "decoratorsBeforeExport": true }],
+    ["@babel/plugin-proposal-decorators", { "version": "legacy" }],
```

Comment:
I tested `{ "version": "legacy" }` with `ember-container-query` as well as 4 private addons. The tests for these addons passed.